README.md

```diff
@@ -322,14 +322,45 @@ You will still need to edit the blocks themselves via the sidebar.
 If you want to make the editing experience the most intuitive, you can enable real-time inplace editing, where an editor
 can change check, links or media by typing or clicking directly on your frontend instead of via fields on the sidebar.
 
-#### Inline text editing ([TODO](https://github.com/collective/volto-hydra/issues/5))
-You will add data attributes to where a blocks text is editable and where text formatting and features are locationed (like links)
-and also subscribe to ```onBlockFieldChanged``` events. This will enable handling fine grained 
-changes to text being edited such as turning text bold or creating a link. Hydra will notice where you have indicated a block field can 
-be clicked on and will automatically make it inplace editable handling typing, shortcuts 
+#### Inline text editing
+
+Now Hydra supports inline editing of text! You can enable this by adding an attribute `data-hydra-node="<<nodeId>>"` to your html nodes.
+The `nodeId` is accessible from `data` object itself once you subscribe to `onEditChange` for realtime data updates (since nodeId is not always available its recommended to use optional chaining). 
+For example if you are using slate to render the text block, then:
+```jsx
```

Comment:
I think in addition to the code it would make it clearer to show an example of what the resulting html looks like

Reply:
Also you need to make it clear that this is just an example of how your code might look if you choose to use react. But it would work similar in other frameworks. or it might even be less confusing to only show the resulting html instead of the react code?